<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="MAVBCP778" ignored="0" name="Default test" tests="19" failures="5" timestamp="2020-08-06T16:48:33 IST" time="168.729" errors="0">
  <testcase name="Initiate the cardAuth_authorize API with valid fields and verify the Expected Result" time="7.461" classname="com.maveric.core.cucumber.CucumberBaseTest"/>
  <testcase name="Initiate the cardAuth_authorize API with valid fields and verify the Expected Result" time="5.104" classname="com.maveric.core.cucumber.CucumberBaseTest">
    <failure type="com.jayway.jsonpath.PathNotFoundException" message="No results for path: $[&amp;apos;access_token&amp;apos;]">
      <![CDATA[com.jayway.jsonpath.PathNotFoundException: No results for path: $['access_token']
at com.jayway.jsonpath.internal.path.EvaluationContextImpl.getValue(EvaluationContextImpl.java:133)
at com.jayway.jsonpath.JsonPath.read(JsonPath.java:187)
at com.jayway.jsonpath.internal.JsonContext.read(JsonContext.java:102)
at com.jayway.jsonpath.internal.JsonContext.read(JsonContext.java:85)
at com.api.service.Base_MoneyMovement.API_clientCredentials(Base_MoneyMovement.java:72)
at com.stepdefs.Transactions_stepdefs.initiate_the_Client_Credentials_API_with_and_get_the_accessToken(Transactions_stepdefs.java:29)
at ✽.Initiate the Client_Credentials_API with 'loan_authorization' and '/apis'  get the accessToken(file:///D:/Jyoti/API/Restassured_WithBaseclass/RestAssured.zip_expanded/./src/test/resources/features/TestCases.feature:10)
]]>
    </failure>
  </testcase> <!-- Initiate the cardAuth_authorize API with valid fields and verify the Expected Result -->
  <testcase name="Initiate the cardAuth_authorize API with valid fields and verify the Expected Result" time="5.171" classname="com.maveric.core.cucumber.CucumberBaseTest">
    <failure type="com.jayway.jsonpath.PathNotFoundException" message="No results for path: $[&amp;apos;access_token&amp;apos;]">
      <![CDATA[com.jayway.jsonpath.PathNotFoundException: No results for path: $['access_token']
at com.jayway.jsonpath.internal.path.EvaluationContextImpl.getValue(EvaluationContextImpl.java:133)
at com.jayway.jsonpath.JsonPath.read(JsonPath.java:187)
at com.jayway.jsonpath.internal.JsonContext.read(JsonContext.java:102)
at com.jayway.jsonpath.internal.JsonContext.read(JsonContext.java:85)
at com.api.service.Base_MoneyMovement.API_clientCredentials(Base_MoneyMovement.java:72)
at com.stepdefs.Transactions_stepdefs.initiate_the_Client_Credentials_API_with_and_get_the_accessToken(Transactions_stepdefs.java:29)
at ✽.Initiate the Client_Credentials_API with 'loan_authorization' and '/api'  get the accessToken(file:///D:/Jyoti/API/Restassured_WithBaseclass/RestAssured.zip_expanded/./src/test/resources/features/TestCases.feature:10)
]]>
    </failure>
  </testcase> <!-- Initiate the cardAuth_authorize API with valid fields and verify the Expected Result -->
  <testcase name="Initiate the Password API with valid fields and verify the Expected Result" time="7.619" classname="com.maveric.core.cucumber.CucumberBaseTest"/>
  <testcase name="Initiate_the_Account_API_with_valid_fields_and_verify_the_Expected_Result" time="9.202" classname="com.maveric.core.cucumber.CucumberBaseTest"/>
  <testcase name="Initiate the GET_Card API with valid fields and verify the Expected Result" time="9.424" classname="com.maveric.core.cucumber.CucumberBaseTest"/>
  <testcase name="Initiate the GET_Card API with valid fields and verify the Expected Result" time="8.979" classname="com.maveric.core.cucumber.CucumberBaseTest"/>
  <testcase name="Initiate_the_Transactions_API_with_valid_fields_and_verify_the_Expected_Result" time="9.225" classname="com.maveric.core.cucumber.CucumberBaseTest"/>
  <testcase name="Initiate_the_Transactions_API_with_valid_fields_and_verify_the_Expected_Result" time="4.586" classname="com.maveric.core.cucumber.CucumberBaseTest">
    <failure type="com.jayway.jsonpath.PathNotFoundException" message="No results for path: $[&amp;apos;access_token&amp;apos;]">
      <![CDATA[com.jayway.jsonpath.PathNotFoundException: No results for path: $['access_token']
at com.jayway.jsonpath.internal.path.EvaluationContextImpl.getValue(EvaluationContextImpl.java:133)
at com.jayway.jsonpath.JsonPath.read(JsonPath.java:187)
at com.jayway.jsonpath.internal.JsonContext.read(JsonContext.java:102)
at com.jayway.jsonpath.internal.JsonContext.read(JsonContext.java:85)
at com.api.service.Base_MoneyMovement.API_clientCredentials(Base_MoneyMovement.java:72)
at com.stepdefs.Transactions_stepdefs.initiate_the_Client_Credentials_API_with_and_get_the_accessToken(Transactions_stepdefs.java:29)
at ✽.Initiate the Client_Credentials_API with 'grant_type' and 'scope'  get the accessToken(file:///D:/Jyoti/API/Restassured_WithBaseclass/RestAssured.zip_expanded/./src/test/resources/features/TestCases.feature:61)
]]>
    </failure>
  </testcase> <!-- Initiate_the_Transactions_API_with_valid_fields_and_verify_the_Expected_Result -->
  <testcase name="Initiate_the_Transactions_API_with_valid_fields_and_verify_the_Expected_Result" time="8.997" classname="com.maveric.core.cucumber.CucumberBaseTest"/>
  <testcase name="Initiate_the_Transactions_API_with_valid_fields_and_verify_the_Expected_Result" time="4.608" classname="com.maveric.core.cucumber.CucumberBaseTest">
    <failure type="com.jayway.jsonpath.PathNotFoundException" message="No results for path: $[&amp;apos;access_token&amp;apos;]">
      <![CDATA[com.jayway.jsonpath.PathNotFoundException: No results for path: $['access_token']
at com.jayway.jsonpath.internal.path.EvaluationContextImpl.getValue(EvaluationContextImpl.java:133)
at com.jayway.jsonpath.JsonPath.read(JsonPath.java:187)
at com.jayway.jsonpath.internal.JsonContext.read(JsonContext.java:102)
at com.jayway.jsonpath.internal.JsonContext.read(JsonContext.java:85)
at com.api.service.Base_MoneyMovement.API_clientCredentials(Base_MoneyMovement.java:72)
at com.stepdefs.Transactions_stepdefs.initiate_the_Client_Credentials_API_with_and_get_the_accessToken(Transactions_stepdefs.java:29)
at ✽.Initiate the Client_Credentials_API with 'grant_type' and 'scope'  get the accessToken(file:///D:/Jyoti/API/Restassured_WithBaseclass/RestAssured.zip_expanded/./src/test/resources/features/TestCases.feature:61)
]]>
    </failure>
  </testcase> <!-- Initiate_the_Transactions_API_with_valid_fields_and_verify_the_Expected_Result -->
  <testcase name="Initiate_the_Transactions_API_with_valid_fields_and_verify_the_Expected_Result" time="10.13" classname="com.maveric.core.cucumber.CucumberBaseTest"/>
  <testcase name="Initiate the sourceAccounts &amp; destination Eligibility API with valid fields and" time="9.207" classname="com.maveric.core.cucumber.CucumberBaseTest"/>
  <testcase name="Initiate the sourceAccounts &amp; destination Eligibility API with valid fields and" time="5.023" classname="com.maveric.core.cucumber.CucumberBaseTest">
    <failure type="com.jayway.jsonpath.PathNotFoundException" message="No results for path: $[&amp;apos;access_token&amp;apos;]">
      <![CDATA[com.jayway.jsonpath.PathNotFoundException: No results for path: $['access_token']
at com.jayway.jsonpath.internal.path.EvaluationContextImpl.getValue(EvaluationContextImpl.java:133)
at com.jayway.jsonpath.JsonPath.read(JsonPath.java:187)
at com.jayway.jsonpath.internal.JsonContext.read(JsonContext.java:102)
at com.jayway.jsonpath.internal.JsonContext.read(JsonContext.java:85)
at com.api.service.Base_MoneyMovement.API_clientCredentials(Base_MoneyMovement.java:72)
at com.stepdefs.Transactions_stepdefs.initiate_the_Client_Credentials_API_with_and_get_the_accessToken(Transactions_stepdefs.java:29)
at ✽.Initiate the Client_Credentials_API with 'grant_type' and 'scope'  get the accessToken(file:///D:/Jyoti/API/Restassured_WithBaseclass/RestAssured.zip_expanded/./src/test/resources/features/TestCases.feature:83)
]]>
    </failure>
  </testcase> <!-- Initiate the sourceAccounts & destination Eligibility API with valid fields and -->
  <testcase name="Initiate the sourceAccounts &amp; destination Eligibility API with valid fields and" time="11.237" classname="com.maveric.core.cucumber.CucumberBaseTest"/>
  <testcase name="Initiate the PDT-PreProcessAPI with valid fields and verify the Expected Result" time="12.586" classname="com.maveric.core.cucumber.CucumberBaseTest"/>
  <testcase name="To verify the PDT-PreProcess API with invalid sourceAccountId" time="12.073" classname="com.maveric.core.cucumber.CucumberBaseTest"/>
  <testcase name="Initiate the PDT-Confirmation API with valid fields and verify the Expected Result" time="14.731" classname="com.maveric.core.cucumber.CucumberBaseTest"/>
  <testcase name="To verify the PDT-Confirmation API with invalid controlFlowId" time="12.495" classname="com.maveric.core.cucumber.CucumberBaseTest"/>
</testsuite> <!-- Default test -->
